### DISCUSSION TITLE
GPT-4 version?

### DISCUSSION THREAD

**Discussion by momobobe** [waylaidwanderer/node-chatgpt-api#103] [Discussions]
Excuse me, afaik GPT-4 is still under development and what Bing uses is still the current version of ChatGPT aka GPT-3.5, with some modifications though. Maybe we should clarify the usage of such terms.

**Comment by waylaidwanderer** [comment]
Bing's AI uses GPT-4 as mentioned in the press conference where they demo'd it. I've also "confirmed" this from purported internal Microsoft sources.
